platforms/unix/vm/sqUnixMain.c

```diff
@@ -1467,7 +1464,8 @@ static int vm_parseArgument(int argc, char **argv)
 
   /* vm arguments */
 
-  if      (!strcmp(argv[0], VMOPTION("help")))		{ usage();		return 1; }
+  if      (!strcmp(argv[0], VMOPTION("help")))		{ usage(0);         /*NOTREACHED*/}
```

Comment:
Good idea. Usually, usage is a fatal error and is not expected to return. But in this case, moving exit out improves readability. Thanks.